### PR TITLE
Collect os_log_bytes_written from information schema.

### PR DESCRIPTION
**Pull request by CharlesJUDITH**
This is done to collect the os_log_bytes_written metric from
infromation_schema.metrics table.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -296,6 +296,13 @@ TBD:
 	hash_index_cells_used
 	additional_pool_alloc
 
+### InnoDB metrics from information_schema
+
+Collected by parsing the output of `SELECT COUNT from INFORMATION_SCHEMA.INNODB_METRICS where name ='os_log_bytes_written';`:
+
+    mysql plugin: Sending value: innodb/os_logs_bytes_written=263167952896
```
Comment (khodin) [inline README.md]: why not being consistent with the name of source metric : os_log_bytes_written
therefore innodb/os_log_bytes_written

File: mysql.py [hunk 2]
```diff
@@ -410,6 +410,11 @@ def fetch_mysql_db_size(conn):
     		stats[row['db_name']] = row['db_size_mb']
 	return stats
 
+def fetch_os_logs_written(conn):
+	result = mysql_query(conn, "SELECT COUNT from INFORMATION_SCHEMA.INNODB_METRICS where name ='os_log_bytes_written';")
```
Comment (khodin) [inline mysql.py:414]: same here fetch_os_log_bytes_written for function name

File: mysql.py [hunk 3]
```diff
@@ -584,6 +589,9 @@ def read_callback():
 	for key in mysql_db_size:
             	dispatch_value('db_size', key, mysql_db_size[key], 'gauge')
 
+	innodb_logs_written = fetch_os_logs_written(conn)
+	dispatch_value('innodb', 'os_logs_bytes_written', innodb_logs_written['COUNT'], 'counter')
```
Comment (khodin) [inline mysql.py]: same for metric name : os_log_bytes_written

File: README.md (hunk 1 above)
Comment (CharlesJUDITH) [reply README.md]: OK looks good to me.

**Review by khodin — APPROVED** [review]

**Pull request merged**